pycbc/events/coinc.py

```diff
@@ -240,6 +240,87 @@ def time_coincidence(t1, t2, window, slide_step=0):
     return idx1.astype(numpy.uint32), idx2.astype(numpy.uint32), slide.astype(numpy.int32)
 
 
+def time_multi_coincidence(times, slide_step=0, slop=.003,
+                           pivot='H1', fixed='L1'):
+    """ Find multi detector concidences.
+
+    Parameters
+    ----------
+    times: dict of numpy.ndarrays
+        Dictionary keyed by ifo of the times of each single detector trigger.
+    slide_steop: float
+        The interval between time slides
+    slop: float
+        The amount of time to add to the TOF between detectors for coincidence
+    pivot: str
+        ifo used to test coincidence against in first stage
+    fixed: str
+        the other ifo used in the first stage coindicence which we'll use
+        as a fixed time reference for coincident triggers. All other detectors
+        are time slid by being fixed to this detector.
+    """
+    # pivots are used to determine standard coincidence triggers, we then
+    # pair off additional detectors to those.
+    def win(ifo1, ifo2):
+        d1 = Detector(ifo1)
```

Comment:
I think this requires an import detector at the beginning of the file, and should be used as detector.Detector(if01)